platform/mds/mds/src/main/java/org/motechproject/mds/builder/impl/EntityBuilderImpl.java

```diff
@@ -121,18 +121,18 @@ private ClassData build(EntityDto entity, List<FieldDto> fields, EntityType type
 
                     // this field is related with id field in entity
                     addProperty(
-                            declaring, idType.getTypeClass(), simpleName + Constants.Util.CURRENT_VERSION,
+                            declaring, idType.getTypeClass(), simpleName + Constants.HistoryTrash.CURRENT_VERSION,
                             null
                     );
 
                     // this field contains information about the schema version of an entity
                     addProperty(
-                            declaring, Long.class.getName(), simpleName + StringUtils.capitalize(Constants.Util.SCHEMA_VERSION_FIELD_NAME), null
+                            declaring, Long.class.getName(), simpleName + Constants.HistoryTrash.SCHEMA_VERSION, null
                     );
                     break;
                 case TRASH:
                     // this field contains information about the schema version of an entity
-                    addProperty(declaring, Long.class.getName(), Constants.Util.SCHEMA_VERSION_FIELD_NAME, null);
+                    addProperty(declaring, Long.class.getName(), Constants.HistoryTrash.SCHEMA_VERSION, null);
```

Comment:
Constants.Util.SCHEMA_VERSION_FIELD_NAME = "schemaVersion" 
Constants.HistoryTrash.SCHEMA_VERSION = "SchemaVersion"
? it will worki with old database